### PR TITLE
feat: implemented single playlist functionality

### PR DESCRIPTION
**Pull request by dev-prabhat**
Check these files

https://github.com/dev-prabhat/super-flex-plus/pull/4/files?file-filters%5B%5D=.js&show-viewed-files=true

### CODE REVIEW

File: src/components/PlaylistForm.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React from "react"
```
Comment (iRohitGaur) [inline src/components/PlaylistForm.js:1]: bug: video is not added to the playlist when a new playlist is created. Please look into it

File: src/context/Playlist-Context.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+           setPlaylist({title:"",description:""})
+       }
+    },[playlistResponse])
+    
+
+    useEffect(()=>{
+       if(playlistVideosResponse !== undefined){
+        setCreatedPlaylists(playlists => playlists.map(playlist => {
+            if(playlist._id === playlistVideosResponse.playlist._id) {
+                playlist.videos = [...playlistVideosResponse.playlist.videos]
```
Comment (iRohitGaur) [inline src/context/Playlist-Context.js:65]: suggestion: this should be a `setState` operation

**Review by iRohitGaur — APPROVED** [review]
Nice work!

Gave some comments. please check them.

Good to merge with updates 👍🏻

File: src/context/Playlist-Context.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+    useEffect(()=>{
+       if(playlistVideosResponse !== undefined){
+        setCreatedPlaylists(playlists => playlists.map(playlist => {
+            if(playlist._id === playlistVideosResponse.playlist._id) {
+                playlist.videos = [...playlistVideosResponse.playlist.videos]  
+            }  
+               return playlist
+        }))
+        console.log(createdPlaylists)
```
Comment (SinchanaMS) [inline src/context/Playlist-Context.js:69]: Please remove the log. Also, line 65 should be a setter, do check.

File: src/pages/Explore/Explore.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import React from "react"
```
Comment (SinchanaMS) [inline src/pages/Explore/Explore.js:1]: Suggestion: You can skip the React import in every file. It works fine if it's in just one of the files too. 
There is an option to disable the react import in the editor. You can look into it.

**Review by SinchanaMS — APPROVED** [review]
The code is neat. The UI looks neat as well. Good work!
Please look into the comments.

**Pull request merged**